src/include/daos_srv/daos_server.h

```diff
@@ -623,6 +624,39 @@ d_rank_t dss_self_rank(void);
 
 unsigned int dss_ctx_nr_get(void);
 
+/* Cache for container root */
+struct tree_cache_root {
+	struct btr_root	btr_root;
+	daos_handle_t	root_hdl;
+	unsigned int	count;
+};
+
+int
+obj_tree_insert(daos_handle_t toh, uuid_t co_uuid, daos_unit_oid_t oid,
+		d_iov_t *val_iov);
+int
+obj_tree_destroy(daos_handle_t btr_hdl);
```

Comment:
All above looks migration internal stuff, not necessary to export them.

Reply:
oh, both rebuild and migration will need this obj_tree.